yesod-core/Yesod/Core/Handler.hs

```diff
@@ -780,6 +781,14 @@ deleteCookie a = addHeaderInternal . DeleteCookie (encodeUtf8 a) . encodeUtf8
 setLanguage :: MonadHandler m => Text -> m ()
 setLanguage = setSession langKey
 
+-- | Set attachment file name.
+--
+-- allow UTF-8 character.
+addContentDispositionFileName :: MonadHandler m => T.Text -> m ()
+addContentDispositionFileName name
+    = addHeader "Content-Disposition" $
+      "attachment; filename*=UTF-8''" `mappend` decodeUtf8 (H.urlEncode True (encodeUtf8 name))
```

Comment:
I haven't seen this syntax before. Can you reference the relevant spec that details how this is supposed to work in a comment?